docs/resources/tutorials/cloud/router_conf_with_forwarding_routes/router_conf_with_forwarding_routes.yaml

```diff
@@ -0,0 +1,32 @@
+version: v3.1
+
+participants:
+
+  - name: ROS_2_Domain_0
+    kind: local
+    domain: 0
+
+  - name: ROS_2_Domain_1
+    kind: local
+    domain: 1
+
+routes:
+
+  - src: ROS_2_Domain_0
+    dst:
+      - ROS_2_Domain_1
+
+  - src: ROS_2_Domain_1
+
+topic-routes:
+
+  - name: /secret
```

Comment:
I'd say that this should be `rt/secret`. In any case, test this example using the compiled DDS Router you have.

Reply:
You're right. I tested it and it only works when I set the topic-routes's name to rt/secret.